Makefile

```diff
@@ -50,9 +50,15 @@ help:
 	@echo "        Create a dummy Puppet module."
 	@echo "    fixtures/python-pypi [base_url=...]"
 	@echo "        Create a PyPI Python repository."
+	@echo "    fixtures/rpm-advisory-bad"
```

Comment:
Why is it bad? If the goal is to make  an incomplete advisory, maybe it makes sense to name it the same way? rpm-advisory-inomplete?

Reply:
It is not really 'incomplete', this advisory will be with same id, version and dates as in 'unsigned' but missing one package from collection. 
Agree to try to re-word it. Will use something like 'rpm-advisory-incomplete-package-list'.